src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -12,35 +12,35 @@
     "power.toomanyorigins.aerial_stability.name": "Hava Dengesi",
-    "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızınız biraz azaltılmış.",
+    "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızın biraz azaltılmış.",
     "power.toomanyorigins.conductive.name": "İletken",
     "power.toomanyorigins.conductive.description": "Yıldırım tarafından çarpıldığında şarjlanırsın.",
     "power.toomanyorigins.stubby_legs.name": "Güdük Bacaklar",
-    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların yüzünden karada daha yavaş yürürsün.",
+    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların yüzünden ayakta daha yavaş yürürsün.",
   
     "effect.toomanyorigins.charged": "Şarjlı",
-    "death.attack.overheat": "%1$s içini fitilledi",
-    "death.attack.overheat.player": "%1$s, %2$s'den kaçmaya çalışırken içini fitilledi.",
-    "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken içini fitilledi.",
+    "death.attack.buildup": "Buildup %1$s için çok fazlaydı",
+    "death.attack.buildup.player": "Buildup, %2$s'den kaçmaya çalışan %1$s için çok fazlaydı",
+    "death.attack.buildup.player.item": "Buildup, %3$s kullanan  %2$s'den kaçmaya çalışan %1$s için çok fazlaydı",
   
     "power.toomanyorigins.hover.name": "Süzül",
-    "power.toomanyorigins.hover.description": "Düşeceğin zaman havada asılı kalabilirsin, bu şüreçte biraz acıkırsın.",
+    "power.toomanyorigins.hover.description": "Düşeceğin zaman havada asılı kalabilirsin, bu şüreçte acıkırsın.",
     "power.toomanyorigins.pollination.name": "Polen Şaçma",
     "power.toomanyorigins.pollination.description": "Eğilmeyerek kemik tozu kullanman bitişik 4 bloğu da etkiler.",
     "power.toomanyorigins.calming_aura.name": "Sakin Aura",
-    "power.toomanyorigins.calming_aura.description": "Çiftleştirdiğin saldırgan olmayan hayvanlar beklemeden tekrar çiftleşebilir..",
+    "power.toomanyorigins.calming_aura.description": "Çiftleştirdiğin oyunculara veya saldırgan canlılara saldıramayan hayvanlar beklemeden tekrar çiftleşebilir.",
     "power.toomanyorigins.stinging_pains.name": "Batma Acısı",
-    "power.toomanyorigins.stinging_pains.description": "Düşman canlıları doğrudan öldürdüğünde acıkırsın.",
+    "power.toomanyorigins.stinging_pains.description": "Düşman canlılara doğrudan hasar verdiğinde acıkırsın.",
     "power.toomanyorigins.unity.name": "Birlik",
     "power.toomanyorigins.unity.description": "Maksimum canın, açlık seviyene bağlı.",
 
     "power.toomanyorigins.affliction.name": "Izdırap",
-    "power.toomanyorigins.affliction.description": "Bazı atılabilirlerin(ok, mızrak vs.) sağlıklı canlılara Wither uygular.",
+    "power.toomanyorigins.affliction.description": "Attığın oklar ve mızraklar canlılara Solma uygular.",
```

Comment:
just says arrows and tridents now instead of saying throwables due to lack of projectile as a word in Turkish